server/src/test/java/org/spine3/server/storage/AbstractStorageShould.java

```diff
@@ -73,15 +72,13 @@ public void tearDownAbstractStorageTest() {
 
     /**
      * Closes the storage and propagates an exception if any occurs.
-     *
-     * @see Throwables#propagate(Throwable)
```

Comment:
`Throwables#propagate` will be removed in the 21.0.

That's why we don't mention it anymore.